src/Resizer.php

```diff
@@ -116,6 +119,11 @@ protected function executeResize(ImageInterface $image, ResizeCoordinatesInterfa
             $imagineOptions['format'] = strtolower(pathinfo($path, PATHINFO_EXTENSION));
         }
 
+        // Fix bug with undefined index notice in Imagine
```

Comment:
I guess that's temporary and should be fixed in Imagine?

Reply:
Yes, pull request is in the works :)

Reply:
https://github.com/avalanche123/Imagine/pull/718